src/content/pages/program.md

```diff
@@ -4,4 +4,56 @@ title: Program
+
+Examples of other ideas for specialist tracks include:
+
+- Something specialised within the data and science space, for example geoscience or life sciences
+- GLAM (Galleries, Libraries Archives and Museums)
+- Information security/cybersecurity
+
+If you’ve run a specialist track before, we encourage you to submit ideas again and also to invite new faces as co-organisers. Succession planning and new ideas are always welcome.
+
+## What to expect when you’re expect(ed to run a specialist track)
+
+PyCon AU runs an integrated Call For Proposals (CFP), where speakers can submit to both the main conference and to the specialist tracks through the same process.
+
+As a specialist track organiser, you will promote the CFP, and help us make sure that the CFP makes its way in front of as many people as possible - including all the ones that you want to have on your track’s stage.
+
+Once the CFP closes, you will participate in the CFP review process, where you’ll need to review all talks submitted to your track, and may (but don’t need to) review talks submitted to the main conference and other tracks too.
+
+You will then select and plan your program, in consultation with the core team and other track organisers. Talks can be moved between the tracks (and the main conference) as needed, after discussion with the presenter, core team, and relevant track organisers.
+
+On the day, you will be the face of your specialist track. You will present opening and closing remarks, introduce speakers, and moderate questions. All of this will happen with the support of the conference and AV volunteers.
+
+All of the other infrastructure (venue, catering, ticketing, marketing, scheduling, sponsorship, AV, volunteers, and all those other fun things) will be handled by the PyCon AU core team. You will be working closely with us, so that we can all make sure your plans turn into reality. (Sadly, this sometimes means we have to say “no”, but hopefully not too often!)
+
+If you’ve got an idea that doesn’t follow the “standard” track-of-talks format, please let us know! We’d love to try new and interesting ideas. As noted above, we’ll need to work together to make sure they’re doable, and that the day runs as smoothly as possible.
+
+It is also important to note that as a specialist track organiser, you’ll be a part of the team working to make the conference happen, and one of the public faces of the conference. This means that we need to be able to trust that you’ll not only follow the code of conduct, but provide a model for the best behaviours we all wish to see in each other.
+
+## How to apply
+
+To propose a track, please email us at [program@pycon.org.au](mailto:program@pycon.org.au) before the end of **Monday, April 28th, 2024** [anywhere on earth (UTC−12)](https://en.wikipedia.org/wiki/Anywhere_on_Earth), and provide us the following information:
```

Comment:
Does the program@pycon.org.au address work and do we have access to it?

Reply:
It does, and we do! I have tested this.